sdks/python/apache_beam/dataframe/pandas_doctests_test.py

```diff
@@ -832,7 +826,6 @@ def test_top_level(self):
             'melt': ['*'],
             'merge': ["df1.merge(df2, how='cross')"],
             'merge_asof': ['*'],
-            'pivot': ['*'],
```

Comment:
I think this should be wont_implement_ok instead, right?

Reply:
Done!